.github/workflows/test.yml

```diff
@@ -187,6 +187,23 @@ jobs:
           name: "${{ github.sha }}-${{ matrix.part }}-race-output"
           path: ./${{ matrix.part }}-race-output.txt
 
+  test-rosetta:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v2
+      - uses: technote-space/get-diff-action@v4
+        id: git_diff
```

Comment:
@marbar3778 does this look correct to you?

Reply:
this part here checks for changes in *.go code